modules/misc/news.nix

```diff
@@ -2029,6 +2029,23 @@ in
           A new module is available: 'programs.foot'.
         '';
       }
+
+      {
+        time = "2021-04-13T07:19:36+00:00";
+        message = ''
```

Comment:
Shouldn't this be conditioned to only show for users of the mbsync module? I guess people who don't wouldn't care about this.